scaleway/data_source_domain_record_test.go

```diff
@@ -0,0 +1,65 @@
+package scaleway
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccScalewayDataSourceDomainRecord_Basic(t *testing.T) {
+	tt := NewTestTools(t)
+	defer tt.Cleanup()
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: tt.ProviderFactories,
+		CheckDestroy:      testAccCheckScalewayDomainRecordDestroy(tt),
+		Steps: []resource.TestStep{
+			{
+				Config: fmt.Sprintf(`
+					resource scaleway_domain_record main {
+						dns_zone = "test-data-source.%s"
+						name     = "www"
+						type     = "A"
+						data     = "1.2.3.4"
+						ttl      = 3600
+					}
+
+					data scaleway_domain_record test {
+						dns_zone  = "${scaleway_domain_record.main.dns_zone}"
+						record_id = "${scaleway_domain_record.main.id}"
+					}
+
+					data scaleway_domain_record test2 {
+						dns_zone = "${scaleway_domain_record.main.dns_zone}"
+						name     = "${scaleway_domain_record.main.name}"
+						type     = "${scaleway_domain_record.main.type}"
+						data     = "${scaleway_domain_record.main.data}"
```

Comment:
Can we retrieve the ttl information?

Reply:
It's possible to retrieve the geoip data?

Reply:
done, I found a bug doing that. Name and type was not exported in the `resourceScalewayDomainRecordRead`